liquigraph-core/src/test/java/org/liquigraph/core/validation/XmlSchemaValidatorTest.java

```diff
@@ -104,11 +104,30 @@ public void parses_changelog_with_explicit_schema() throws Exception {
         assertThat(validator.validateSchema(asNode("changelog/changelog-with-schema-location.xml"))).isEmpty();
     }
 
+    @Test
+    public void parses_changelog_with_parameters() throws Exception {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00100)